recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+
+build:
+    number: 0
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - cython >=0.15
+    - numpy x.x
+    script $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+    
+  run:
+    - python >=2.7
```

Comment:
I have an example of a recipe which has a python requirement, but which needs to run against the python it was built with (which I don't think this recipe does). You can see it here: https://github.com/SciTools/conda-recipes-scitools/blob/master/cartopy/meta.yaml

Essentially, you need to put the python requirement in both build and run, and the build requirement also needs a plain Python requirement so that the version it was actually built with gets baked in.

Slightly awkward syntax I think...